test/Triton/verify-make-range.mlir

```diff
@@ -0,0 +1,35 @@
+// RUN: triton-opt --split-input-file %s --verify-diagnostics
```

Comment:
nit: can we make this file `invalid.mlir` so that we add all our verifier tests in there for this dialect?

Reply:
Actually I would rather use separate files, because
 - you get fewer merge conflicts, and
 - you can run just the tests you want (faster / less noise).

Any benefits to having it all in one big file?

Reply:
fair enough, I was mostly suggesting it as this is the convention followed in MLIR upstream overall but I don't have a strong opinion and your points are valid.